config/initializers/delayed_job.rb

```diff
@@ -2,6 +2,5 @@
 Delayed::Worker.max_attempts = 10
 Delayed::Worker.max_run_time = 5.minutes
 Delayed::Worker.destroy_failed_jobs = false
-Delayed::Worker.logger = Logger.new(Rails.root.join('log', 'delayed_job.log'))
```

Comment:
does DJ also need a `SemanticLogger[DelayedJob]` here?

Reply:
semantic_logger_rails has automatic support for DJ